includes/admin/class-wc-stripe-admin-notices.php

```diff
@@ -58,6 +65,11 @@ public function admin_notices() {
 		// All other payment methods.
 		$this->payment_methods_check_environment();
 
+		// Subscription related checks.
+		if ( class_exists( 'WC_Subscriptions' ) && class_exists( 'WC_Subscription' ) && version_compare( WC_Subscriptions::$version, '2.2.0', '>=' ) ) {
```

Comment:
I have moved a helper method with this check [here](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3864) so we can use it in non-gateway classes.